src/components/dashboard-settings/sync-apps/sync-apps.directive.js.coffee

```diff
@@ -89,108 +40,73 @@ module.directive('dashboardSettingSyncApps', ($templateCache, $log, $http, $filt
-
-          switch connector.status
-            when 'SUCCESS'
-              status = if modalDisplay then "Synced at #{date}" else "is synced - #{date}"
-            when 'FAILED'
-              status = if modalDisplay then "Previous sync #{date}" else "is not synced - previous sync #{date}"
-            when 'DISCONNECTED'
-              status = if modalDisplay then "Previous sync #{date}" else "is not connected - previous sync #{date}"
-            when 'DISABLED'
-               status = if modalDisplay then "Previous sync #{date}" else "synchronisation is disabled - previous sync #{date}"
-            # "RUNNING" case should imply isSyncing==true...
-
-        else
-          switch connector.status
-            when 'FAILED'
-              status = if modalDisplay then "Sync failed" else "is not synced - sync failed"
-            when 'NOT SYNCED'
-              status = if modalDisplay then "Never synced" else "is not synced - never synced"
-            when 'DISCONNECTED'
-              status = if modalDisplay then "Not connected" else "is not synced - not connected"
-            when 'DISABLED'
-               status = if modalDisplay then "Synchronization disabled" else "synchronisation is disabled"
-
-        status = if modalDisplay then "#{status}" else "<strong>#{name}</strong> #{status}"
-        status = $sce.trustAsHtml(status) unless _.isEmpty(status) || _.isEmpty(name)
-        return status
+        $http.post(ImpacRoutes.organizations.appInstancesSync(scope.orgUID)).then( (resp) -> processAppInstancesSync(resp.data) )
 
       scope.triggerSyncAlertsModal = ->
         modalInstance = $modal.open({
           animation: true
           size: 'md'
           templateUrl: 'alerts.tmpl.html'
           appendTo: angular.element('impac-dashboard')
-          controller: ($scope, connectors, methods) ->
-            $scope.failedConnectors = connectors.failed
-            $scope.disconnectedConnectors = connectors.disconnected
-            $scope.disabledConnectors = connectors.disabled
-            $scope.successfulConnectors = connectors.successful
-            $scope.formatStatus = methods.formatStatus
-            $scope.expandListItemOnClick = (e) ->
-              list = $document.find('#sync-apps-modal .modal-list')
-              listItem = angular.element(e.target)
-              # Showing only one listItem message at a time.
-              list.find('.message').addClass('ng-hide')
-              listItem.find('.message').removeClass('ng-hide')
-              # Calculates the top offset of the listItem relative to the list.
-              list.animate({ scrollTop: listItem.offset().top - list.offset().top }, 500)
-              return
+          controller: ($scope, connectors) ->
+            $scope.connectors = connectors
+            $scope.expandListItemOnClick = (connector) ->
+              return unless connector.message
+              connector.showMessage = !!!connector.showMessage
```

Comment:
Thinking about this further, I don't think I mind it as it would be consistent with needing `connector.showMessage` to be `true` in the case of assigning to a variable